test/integration/common.py

```diff
@@ -1,12 +1,22 @@
+"""Miscellaneous utilities used in the integration tests."""
+
 from enum import Enum
 
 
 class NighthawkException(Exception):
+  """Base exception class for Nighthawk's python code.
+
+  Args:
```

Comment:
Class docstring doesn't really have Args section, since classes don't take arguments. We would typically put that in the docstring of the __init__ method, but that one isn't needed here.

We can just say:

```py
class NighthawkException(Exception):
  """Base exception class for Nighthawk's python code."""
```